server/plugin.go

```diff
@@ -57,6 +59,22 @@ func (p *Plugin) OnActivate() error {
 	}
 	p.i18nBundle = i18nBundle
 
+	jitsiBot := &model.Bot{
+		Username:    "jitsi",
+		DisplayName: "Jitsi",
+		Description: "A bot account created by the jitsi plugin",
+	}
+	options := []plugin.EnsureBotOption{
+		plugin.ProfileImagePath("assets/icon.png"),
```

Comment:
The icon that we have is an `svg`

Reply:
I think is ok to have a generate png version, but you need to add that to the commit

Reply:
As far as I know a profile picture has to be an `png`, not `svg`.

Reply:
Yes, `svg` profile picture was not accepted. But I forgot to add it to the commit :facepalm: This is fixed!